src/data/publications.ts

```diff
@@ -12,7 +12,7 @@ export const ResearchTopics = [
   'social',
   'human-ai-interaction',
 ] as const
-export type ResearchTopic = (typeof ResearchTopics)[number]
+export type ResearchTopicType = (typeof ResearchTopics)[number]
```

Comment:
Why did you add `Type` at the end? Was there a conflicting name?

Reply:
Earlier in the file, there was a `export type PublicationType = (typeof PublicationTypes)[number]` line, so I renamed `ResearchTopic` to `ResearchTopicType` in order to maintain consistency. Plus, I think `ResearchTopicType` is a more accurate name.